src/components/autocomplete/js/autocompleteController.js

```diff
@@ -641,9 +641,15 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
       $mdUtil.nextTick(function () {
```

Comment:
@winniehell - please incorporate my suggestions ^^ and test. This will make the merged code better moving forward.